infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -575,6 +628,38 @@ private void handleAllocation(CorfuPayloadMsg<TokenRequest> msg, ChannelHandlerC
                 new TokenResponse(token, backPointerMap.build())));
     }
 
+    /**
+     * This method handles the request of streams address maps.
+     * It computes the addresses requested for each stream
+     * and returns this map under a StreamsAddressResponse message.
+     */
+    @ServerHandler(type = CorfuMsgType.STREAMS_ADDRESS_REQUEST)
+    private synchronized void handleStreamsAddressRequest(CorfuPayloadMsg<StreamsAddressRequest> msg,
+                                             ChannelHandlerContext ctx, IServerRouter r) {
+        List<StreamAddressRange> streamsRangesRequested = msg.getPayload().getStreamsRanges();
+        Map<UUID, StreamAddressSpace> streamsAddressSpaceMapToReturn = new HashMap<>();
```

Comment:
let's invent a better name for this variable :)

Reply:
done.